src/execution/compiler/operator/aggregate_translator.cpp

```diff
@@ -269,6 +269,11 @@ void AggregateBottomTranslator::GenHashCall(FunctionBuilder *builder) {
   for (uint32_t term_idx = 0; term_idx < op_->GetGroupByTerms().size(); term_idx++) {
     hash_args.emplace_back(GetGroupByTerm(agg_values_, term_idx));
   }
+  // TODO(Amadou): In case there is no group by term, we can actually bypass the hash table.
+  // For now, I am passing in a constant hash value.
+  if (hash_args.empty()) {
+    hash_args.emplace_back(codegen_->IntToSql(0));
+  }
```

Comment:
We talked about how to solve this. You'll have to undo this change in the new PR.